plugins/grid-bookmark/src/GridBookmarkWidget/components/GridBookmarkWidget.tsx

```diff
@@ -43,8 +39,7 @@ const GridBookmarkWidget = observer(function ({
         <DeleteBookmarks model={model} />
       </div>
       <Alert severity="info">
-        Click or double click the <strong>label</strong> field to notate your
-        bookmark.
+        Double click the <strong>label</strong> field to annotate your bookmark.
```

Comment:
seemed like i could only do double click on my computer(?)

Reply:
It's one click to select the input field and then you just start typing, without the dialog. It works on chrome and firefox on this branch

Reply:
The "Alert" message seems to indicate (to me) that a single click has the same action as a double click. I was not aware of single click + type in the field, and no cursor appears sort of to help confirm that so it's sort of hard to tell otherwise

Reply:
That's functionality that was in the previous iteration bookmarks widget (single click and edit in-line). Agree that the message could be clearer!